src/ui/components/uicontroller/UiContainer.tsx

```diff
@@ -188,10 +190,28 @@ export const UiContainer = (props: UiContainerProps) => {
   const [pip, setPip] = useState(false);
   const [adInProgress, setAdInProgress] = useState(false);
   const [adTapped, setAdTapped] = useState(false);
+  const [tvEventHandlerAvailable, setTvEventHandlerAvailable] = useState(true);
   const appStateSubscription = useRef<any>(null);
   const _menus = useRef<MenuConstructor[]>([]).current;
   const player = props.player;
 
+  if (Platform.OS === 'ios' && Platform.isTV) {
```

Comment:
Can do the same trick here:
```suggestion
  const [tvEventHandlerAvailable, setTvEventHandlerAvailable] = useState(Platform.OS === 'ios' && Platform.isTV);
  const appStateSubscription = useRef<any>(null);
  const _menus = useRef<MenuConstructor[]>([]).current;
  const player = props.player;

  if (tvEventHandlerAvailable) {
```